tests/sentry/release_health/test_metrics_sessions_v2.py

```diff
@@ -84,3 +86,8 @@ def test_sessions_metrics_equal_num_keys(self):
                 rollup=interval_days * 24 * 60 * 60,  # days to seconds
             )
             assert len(errors) == 0
+
+
+def _session_groupby_powerset() -> Iterable[str]:
+    keys = ["project", "release", "environment", "session.status"]
+    return chain.from_iterable((combinations(keys, size)) for size in range(len(keys) + 1))
```

Comment:
Nice! You could probably convert this function to a [generator](https://peps.python.org/pep-0255/):

```suggestion
    for size in range(len(keys) + 1):
        yield from combinations(keys, size)
```